packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -191,7 +192,12 @@ export abstract class CosmosSdkBaseAdapter<T extends CosmosSdkChainId> implement
           })),
         }))
 
-        return { ...data, delegations, redelegations, undelegations, rewards }
+        const assets = data.assets.map<CosmosSDKToken>((asset) => ({
+          amount: asset.amount,
+          denom: asset.denom,
```

Comment:
Can we redeclare `generateAssetIdFromOsmosisDenom` in lib and make this `assetId` for parity with `EVMBaseAdapter`? Lib shouldn't speak token addresses / denoms but rather all CAIP